plugins/path_managers/sspi.c

```diff
@@ -473,7 +506,15 @@ static void sspi_send_addr(void *data, void *user_data)
          *      subflows exist for the addr in question, meaning there
```

Comment:
Looks like this comment should be deleted, since the change below uses real values.

Reply:
(github web ui wouldn't let me annotate the first line of this comment, it was too early in the diff context)